### ISSUE TITLE
No WCAG success criteria for users with Speech impairments

### ISSUE THREAD

**Issue by rahulpundir019** [w3c/wai-website#521]
### Resource URL

https://www.w3.org/WAI/WCAG22/Understanding/

### Add a description

Recently I was using a service where you can input your data only using your voice so when I checked WCAG guidelines, there is no Success criteria considering this as a failure use case so just want to know why there is no guidelines for these type of use cases.
